server/graphql/v2/query/collection/ExpensesCollectionQuery.ts

```diff
@@ -217,7 +217,10 @@ const ExpensesCollectionQuery = {
       where['createdAt'] = where['createdAt'] || {};
       where['createdAt'][Op.lte] = args.dateTo;
     }
-    if (args.payoutMethodType) {
+
+    if (args.payoutMethodType === 'CREDIT_CARD') {
+      where[Op.and].push(sequelize.literal(`("VirtualCardId" IS NOT NULL)`));
```

Comment:
Any argument in favor of a literal rather than standard sequelize operators? I feel that the latter is more consistent with what we do elsewhere.

```suggestion
      where[Op.and].push({ VirtualCardId: { [Op.not]: null } });
```